ansible/roles/util-cfg-service/templates/ui/de.properties.j2

```diff
@@ -130,6 +130,7 @@ org.iplantc.discoveryenvironment.workspace.defaultAppCategories      = {{ apps.u
 org.iplantc.discoveryenvironment.workspace.defaultBetaAppCategoryId  = {{ apps.beta_category }}
 org.iplantc.discoveryenvironment.workspace.defaultHpcAppCategoryId   = {{ apps.hpc_category }}
 org.iplantc.discoveryenvironment.workspace.defaultTrashAppCategoryId = {{ apps.trash_category }}
+org.iplantc.discoveryenvironment.workspace.ontologyAttrs             = {{ apps.ontology_attrs }}
```

Comment:
Ha, I just did that in my other PR.  Not sure why I spaced out on this PR.  Thanks for catching!